pkg/api/v1alpha1/awsiamconfig_webhook_test.go

```diff
@@ -35,11 +35,45 @@ func TestValidateUpdateAWSIamConfigSuccess(t *testing.T) {
 	g.Expect(aiNew.ValidateUpdate(&aiOld)).To(Succeed())
 }
 
+func TestValidateCreateAWSIamConfigSuccess(t *testing.T) {
+	aiNew := awsIamConfig()
+
+	g := NewWithT(t)
+	g.Expect(aiNew.ValidateCreate()).To(Succeed())
+}
+
+func TestValidateCreateAWSIamConfigFail(t *testing.T) {
+	aiNew := awsIamConfig()
+	aiNew.Spec.AWSRegion = ""
+
+	g := NewWithT(t)
+	g.Expect(aiNew.ValidateCreate()).NotTo(Succeed())
```

Comment:
could we validate the error message with `MatchError(ContainSubstring(--))`?

Reply:
Good idea